src/DrTests/CompiledMethod.extension.st

```diff
@@ -6,6 +6,12 @@ CompiledMethod >> asResultForDrTest [
 	^ DTTestLeafNode content: self
 ]
 
+{ #category : #'*DrTests' }
+CompiledMethod >> asResultForDrTestCoverage [
+
+	^ DTTestCoverageLeafNode content: self
+]
+
```

Comment:
Because this is only called in one place so far (`DTTestCoverageResult>>#buildTreeByClass:`), I think this method is not yet necessary.
```suggestion
```